src/components/Preview/Preview.tsx

```diff
@@ -73,6 +74,8 @@ class Preview extends React.Component<Props & CollectedProps, State> {
     }
     try {
       isDCLInitialized = true
+      // tslint:disable-next-line: no-eval
+      eval('window.devicePixelRatio = 1')
```

Comment:
is `eval` necessary here? why not just doing `(window as any)` or even typing the window?

Reply:
mm this is coming from a fix from pato (because i pulled from master), i guess it should be the same, but i'm not sure how to test that

Reply:
Welp 🤸‍♂️